### PR TITLE
3.0 updates

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,111 @@
...
+</script>
 ```
 
-To use an icon from one of these sets, first prefix your `iron-icon` with the icon set name, followed by a colon, ":", and then the icon id.
+To use an icon from one of these sets, first prefix your `iron-icon` with the
+icon set name, followed by a colon, ":", and then the icon id.
 
 Example using the directions-bus icon from the maps icon set:
 
 ```html
```
Comment (notwaldorf) [inline README.md]: This bit of the readme isn't needed, right? Either it's in the docs that we link to, or its in the proper demo section

File: communication-icons.js [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+found at http://polymer.github.io/PATENTS.txt
+*/
 import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
+
 const $_documentContainer = document.createElement('template');
 $_documentContainer.setAttribute('style', 'display: none;');
 
-$_documentContainer.innerHTML = `<iron-iconset-svg name="communication" size="24">
+$_documentContainer.innerHTML =
```
Comment (notwaldorf) [inline communication-icons.js]: We should use the `html` function for consistency, here and everywhere, i.e:
```
import {html} from '@polymer/polymer/lib/utils/html-tag.js';
const template = html``;
document.body.appendChild(template.content);
```

File: README.md (hunk 1 above)
Comment (bicknellr) [reply README.md]: I've removed most of the docs here and just left the links.

File: communication-icons.js (hunk 2 above)
Comment (bicknellr) [reply communication-icons.js]: Was this the comment about using `html` that we were talking about leaving alone?

Comment (notwaldorf) [reply communication-icons.js]: yah but only if you think makes sense. if it looks like it's already being downloaded, then consistency maybe wins?

Comment (bicknellr) [reply communication-icons.js]: iron-icon (imported by this file) [imports `html`](https://github.com/PolymerElements/iron-icon/blob/a805f91aa94a575ab094807f26b44a5267a80b87/iron-icon.js#L16), so I'll switch these.

Comment (bicknellr) [reply communication-icons.js]: These are all updated now.

**Pull request merged**